Annotation/Annotation.php

```diff
@@ -24,4 +24,9 @@ abstract class Annotation
      * @var string
      */
     public $project;
+
+    /**
+     * @var Gordalina\MixpanelBundle\Annotation\Expression
```

Comment:
you should remove this type so you can simplify your `condition` usage like this : https://symfony.com/doc/4.1/routing/conditions.html

(`condition="method...`)